node/pkg/cni/token_watch.go

```diff
@@ -162,6 +154,21 @@ func (t *TokenRefresher) Run() {
 	}
 }
 
+func (t *TokenRefresher) getSleepTime(nextExpiration *time.Time) time.Duration {
+	now := time.Now()
+	sleepTime := nextExpiration.Sub(now) / t.defaultRefreshFraction
+	if duration, err := time.ParseDuration(os.Getenv("CNI_TOKEN_REFRESH_DURATION")); err == nil {
```

Comment:
I think we should refactor this so that if there is an error, we log an Error log indicating such. We want to avoid users setting this to an invalid value and not seeing any feedback.